### DISCUSSION TITLE
purpose/setting of device_url in configuration.yaml?

### DISCUSSION THREAD

**Discussion by jeffalperin** [helgeerbe/picframe#237] [Q&A] [answered]
I just updated to the latest picframe version, but am unsure of how (or if) to set the variable `device_url` which is found in the `mqtt` section of `configuration.yaml`.  I have `use_http` set to `True`.  The comment says to set `device_url` to the "picframe config page"; is this `http://[IP of raspberry pi on my local network]:9000`?  How is `device_url ` being used?  At the moment, I just have it set to the null string, and it seems to be working fine with Home Assistant and on its own.  Thanks for clarifying!  --Jeff

**Comment by helgeerbe** [comment, marked as answer]
Hi @jeffalperin,

have a look at https://developers.home-assistant.io/blog/2021/10/26/config-entity/ .

You find the device_url in home assistant, if you open picframe's device configuration panel. Click on "visit device" and picframe's webpage opens.
![2022-01-27 19_29_59-Configuration - Home Assistant](https://user-images.githubusercontent.com/59169507/151421746-3d34695e-5b3d-4bdb-8dae-9212e36b9cc0.png)

In simple words device_url is the url you use to open picframe's webpage.

**Comment by jeffalperin** [comment]
Got it; thank you!
